packages/core/src/odata/common/filter/filter-function-base.ts

```diff
@@ -0,0 +1,97 @@
+/* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
+
+import { EdmTypeShared } from '../edm-types';
+import { EntityBase, ODataVersionOf } from '../entity';
+import { Field, FieldType } from '../selectable';
+import { Filter } from './filter';
+
+/**
+ * Data structure to represent OData V2 filter functions.
```

Comment:
Is this true? If so this class and the related ones should not be in the common folder but in the V2 specific one.

Reply:
No it's not. It is a left over from before.